release.Jenkinsfile

```diff
@@ -88,7 +88,11 @@ try {
                         }
                     
                         commitID = sh (script: "git rev-parse origin/${params.RELEASE_BRANCH}", returnStdout: true).trim()
-                        configureBuilds(commitID)
+                        configureBuilds()
+
+                        withCredentials([usernamePassword(credentialsId: env.DOCKER_CREDENTIALS, passwordVariable: 'pwd', usernameVariable: 'uname')]) {
```

Comment:
As far as I see you're only pulling images and not pushing anything. For that, you don't need to login.